src/scripts/constants.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class _constants:
+
+    _DEFAULT_TEST_COUNT = 100 # The Default value of --test-count option
+    _DEFAULT_SEED = 10000 # The default value of random seed
+    _TEST_CASE_IN_A_FILE = 700 # The number of test cases before start new file
+    _OPERAND_COUNT_MAX = 11 # The --operand-count option's maximum value
+    _OPERAND_COUNT_MIN = 2 # The --operand-count option's minimum value
+    _MAX = (1 << 53) - 1 # The max value of random number (in JS Number.MAX_SAFE_INTEGER)
+    _MIN = - 1 * _MAX  # The min value of random number (in JS Number.MIN_SAFE_INTEGER)
+    _BITMAX = (1 << 31) - 1 # The max value of random number for bitwise operations
+    _BITMIN = - _BITMAX - 1 # The min value of random number for bitwise operations
+    _BITMAX_EXPOSANT = 31 # Maximum number for safely shifting an integer in python to
+    # be precise for JS 32 bit numbers
+    _BITMIN_EXPOSANT = 0 # Python can not handle negative right operand for shifting
```

Comment:
The comment is valid, but strange...

However in python

```
>>> 5 << -4
ValueError: negative shift count
>>> 5 << (32 - 4)
1342177280
```

In JS:

```
>>> 5 << (- 4)
1342177280
```

So it's possible to convert negative shift count to positive one.

I'd not use this constant. (This change may affect other PR's) but with this trick, the negative shift count could be tested as well.